### DISCUSSION TITLE
Landsat 9 Data Availability [Now available]

### DISCUSSION THREAD

**Discussion by AllisonVincent** [microsoft/PlanetaryComputer#42] [Data request]
Hello,

Are there plans to add Landsat 9 imagery to the Planetary Computer Data Catalog? If so, can you provide an estimate as to when this will be available?

Thank you.

~Allison

**Comment by TomAugspurger** [comment]
Landsat 9 assets are now available in the [landsat-c2-l2](https://planetarycomputer.microsoft.com/dataset/landsat-c2-l2) collection, which also includes assets from Landsat 4, 5, 7, and 8.

---

(original reply)

Landsat-9 assets are currently available through blob storage, in the same storage account / container as Landsat 8 C2-L2 (landsateuwest/landsat-c2). So if you know the paths of the assets you want, you can get a SAS token and get / list the files you need.

We're currently working on generating STAC items and adding it to the STAC API. Hopefully ready within the next month or so!

**Reply by AllisonVincent** [reply]
That all sounds great! Thank you very much.

**Reply by TomAugspurger** [reply]
Late followup, but these Landsat 9 assets are now available in the [landsat-c2-l2](https://planetarycomputer.microsoft.com/dataset/landsat-c2-l2) collection, which also includes assets from Landsat 4, 5, 7, and 8.

You can use the `platform` on the STAC items to select a specific platform.

**Reply by AllisonVincent** [reply]
Thank you for the follow up! The data collections you all have put together are great and have been working very smoothly.
